lib/components/ManageUI.jsx

```diff
@@ -68,14 +67,12 @@ const ManageHeader = () => {
 
         <p className='text-accent-1 text-base sm:text-2xl max-w-3xl'>Version 3.1.0</p>
 
-        <Button
-          size='base'
-          color='secondary'
-          className='mt-6'
+        <a
```

Comment:
Should this be `<ButtonLink>`?

Reply:
The next.js router wasn't having any of it. It would probably make sense to have `<ButtonLink>` know about external links.